src/mobile/android/app/BUCK

```diff
@@ -45,12 +45,12 @@ android_library(
 
 android_build_config(
     name = "build_config",
-    package = "trinity",
+    package = "com.trinity-mobile",
```

Comment:
Was this an automated change? We should leave it alone to avoid breaking something